telemetry/generated/web.ts

```diff
@@ -4,8 +4,8 @@
 
 // AUTOGENERATED BY glean_parser v8.1.1. DO NOT EDIT. DO NOT COMMIT.
 
-import EventMetricType from "@mozilla/glean/private/metrics/event";
 import StringMetricType from "@mozilla/glean/private/metrics/string";
+import EventMetricType from "@mozilla/glean/private/metrics/event";
```

Comment:
I swear these two generated files keep flipping back and forth even though they are supposed to be being ignored by eslint. A problem for another day...